RNAwesomeCardIO.podspec

```diff
@@ -0,0 +1,19 @@
+require 'json'
+package = JSON.parse(File.read('package.json'))
+
+Pod::Spec.new do |s|
+  s.name             = "RNAwesomeCardIO"
+  s.version          = package["version"]
+  s.summary          = package["description"]
+  s.requires_arc     = true
+  s.license          = package["license"]
+  s.homepage         = package["homepage"]
+  s.authors          = package["author"]
+  s.source           = { :git => package["homepage"] }
```

Comment:
Looking at another open source projects, it's good practise to also include the git tag:
```
s.source             = { :git => s.homepage, :tag => "#v{s.version}" }
```